organization/_posts/2022/2022-01-20-coordination.md

```diff
@@ -0,0 +1,214 @@
+
+After this meeting outgoing conveners will be liberated from the mailing list, but we look forward to your continuing involvement in HSF and the WGs.
+
+### Data Analysis
+
+Review:
+- metadata in analysis
+- analysis software in wider community
+- analysis benchmarks
+- workflow management tools
+
+Preview:
+- reach out better to analysis people
+- another workflow management meeting
+- systematic uncertainties in analysis
+- workshop on analysis ecosystems
+
+TODO - Post paper to arXiv! This paper was a really great idea to give a concrete outcome to the WG's activity. When appropriate other WGs should consider the same as the HSF brings a unique pan-experiment perspective.
+
+### Detector Simulation
+
+WG will be involved in organising the GPU R&D community meeting.
+
+Geometry is also important for reconstruction, so this is a cross-over topic between the two WGs.
+
+
+### Reconstruction and Software Triggers
+
+Are there concrete plans in the training? Should start that discussion soon and try and identify the topics. Training is needed in how to convert reco algorithms, which is rather bold and broad.
+
+There would be a link to the Conditions DB activity here as well.
+
+### PyHEP
+
+Suggestion to gather all of the didactic material presented through PyHEP in one place so that people could browse and pick from them (even without (re)formatting things!).
+- Note that we have <https://github.com/hsf-training/PyHEP-resources> linked from the WG webpage and we could either add "tutorials" there and/or have a section in the WG webpage. To be thought about.
+
+What about Julia? Should we organise activities still via PyHEP or separately. Probably keep it under the PyHEP umbrella for now, which is beneficial for gathering interest in this language in HEP.
+
+### Software Tools and Packaging
+
+Good idea to look at new C++ features, a la module of the month.
+
+### Event Generators
+
+Benjamin Morgan: use of julia for generators as in simulation could be a possible topic? 
+josh: for this to work you have to convince all generator authors... To make this work it would require convincing lots of people, the code base it already fragmented in Fortran, C++, Python. Idea is good but in practice it may be difficult. 
+Jim Pivarski: Just a note: in some syntactical ways, Julia is a shorter jump from Fortran than C++ is. In other fields (non-HEP), Julia is replacing Fortran, so generators, which still have more affinity for Fortran, is not inconceivable.
+Benjamin morgan: At least on the detector sim side, it’s not just use of Python/Julia for implementation, but also “good practices” for binding existing C/C++/Fortran libraries to these. That’s a slightly different scope, but can be a good bridge/intro. 
+
+Liz: Any discussion of neutrino generators? 
+Josh: it's on our radar.
+Liz: Fermilab put a proposal to do generators for neutrinos. If succeeded, definitely I'd nominate someone to give a talk on that. 
```

Comment:
Need to add some paragraph breaks here, between each speaker. Also "josh" -> "Josh".

Reply:
done